.github/workflows/commitlint.yml

```diff
@@ -9,10 +9,8 @@ jobs:
       - uses: actions/checkout@v2.3.1
         with:
           fetch-depth: 0
-      - run: npm install --save-dev @commitlint/cli
-      - run: |
-          if [[ "$(gh pr view ${{ github.event.pull_request.number }} --json commits --jq '.[][0].messageHeadline')" == vim-patch* ]];then
-            npx commitlint --from HEAD~1 --to HEAD --verbose --help-url https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md#commit-messages --config .github/workflows/commitlint.config_patch.js
-          else
-            npx commitlint --from HEAD~1 --to HEAD --verbose --help-url https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md#commit-messages --config .github/workflows/commitlint.config.js
```

Comment:
was it previously only checking the last commit, instead of everything after the "merge base"?

we might want to start with that as well.

Reply:
No, it was checking all commits that was made on the pull request.

Reply:
nah